static/app/views/eventsV2/savedQuery/index.tsx

```diff
@@ -457,10 +457,4 @@ const IconUpdate = styled('div')`
   background-color: ${p => p.theme.yellow300};
 `;
 
-const AddToDashboardButton = styled(Button)`
-  span {
-    height: 38px;
-  }
-`;
-
```

Comment:
This special style here was causing problems w/ the new button styles.

Before:
<img width="451" alt="AddToDashboardButton--b" src="https://user-images.githubusercontent.com/44172267/150042322-73f73331-d3ac-43b8-b8de-23325d2be41f.png">

After:
<img width="442" alt="Screen Shot 2022-01-18 at 4 46 53 PM" src="https://user-images.githubusercontent.com/44172267/150042412-5c3c842d-fc15-4489-b516-07fbf597ecf1.png">